pathways/simulation.py

```diff
@@ -97,10 +103,14 @@ def simulation(
     add_pest = get_pest_function(config)
     is_inspection_needed = get_inspection_needed_function(config)
     sample = get_sample_function(config)
-
+    if detailed:
+        details = [[], []]
```

Comment:
Perhaps `stem_details` and `inspected_stem_details` would be better here. We already have a flood of variables here anyway and the indexing in the long stretch of code is hard to check. Clear names won't need any further comment here.

Reply:
Something like this?
```
    if detailed:
        stem_details = []
        inspected_stem_details = []
        details = [stem_details, inspected_stem_details]
```

Reply:
Yes. You don't even need the

```
details = [stem_details, inspected_stem_details]
```

line because you don't need details until you are assigning them to the return value, then you can just do 

```
simulation_results.details = [stem_details, inspected_stem_details]
```